app.json

```diff
@@ -9,7 +9,10 @@
                 "postdeploy": "python manage.py setup_dev"
             },
             "env": {
-                "SELF_CAPTURE": true
+                "SELF_CAPTURE": true,
+                "SECRET_KEY": {
+                    "generator": "secret"
```

Comment:
Heroku, you're super cool. :heart: 

You'd need to do write lambdas to achieve the same thing in cloudformation (trust me, I did the research).